Sources/Valet/SecureEnclaveError.swift

```diff
@@ -0,0 +1,82 @@
+//  Created by Allison Moyer on 5/28/21.
+//  Copyright © 2021 Square, Inc. All rights reserved.
+//
+//  Licensed under the Apache License Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing software
+//  distributed under the License is distributed on an "AS IS" BASIS
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+import Foundation
+import LocalAuthentication
+
+public enum SecureEnclaveError: Int, CaseIterable, CustomStringConvertible, Error, Equatable {
+    /// Access to the secure enclave was not attempted because it was not available (due to biometrics not being available or enrolled
+    /// or being locked out, or the authentication context being invalidated)
+    case couldNotAccess
+
+    /// Access to the secure enclave was not attempted because the user failed to provide valid credentials.
+    case authenticationFailed
```

Comment:
Let's fold this into `userCancelled` – see my other comment about it (and let's consolidate discussion on the other thread).

Reply:
👍